translation/en/RedirectedResources/assets/abdata/communication/info_00/communication_20/translation.txt

```diff
@@ -1,22 +1,22 @@
-//「きゃっ！　あの、ここは女子トイレですよ？」=
-//「きゃああっ！　み、見ないでください！」=
-//「その……急いでいるので失礼します」=
-//「……なんですか？」=
-//「ご、ごめんなさい……今、友達に呼ばれていて……し、失礼します」=
-//「すー……すー……」=
-//「えっ！？　……あ。そ、そっち向いてください！」=
-//「うぅ、あの、あんまり見ないでください」=
-//「あ、どうも……あなたも、屋上が好きなんですか？」=
-//「あ、あなたも本を借りに来たんですか？」=
-//「あ、あれ？　き、奇遇ですね」=
-//「えっと……何をするかも、着くまでナイショなんですか？」=
-//「あの、さすがにそこまでは……」=
-//「い、いつも話しかけて来てくれて、嬉しいです」=
-//「あ、あの、私とで良ければ……な、なんでもありません！」=
-//「あっ、どうも。えっと、見学ですか？」=
-//「あ、あれ？　今日はコイカツ部もここで活動を……？」=
-//「部活中……ですか？」=
-//「お昼……ですね。少しお腹が空きました」=
+「きゃっ！　あの、ここは女子トイレですよ？」="Hey! Isn't this the girls' restroom?"
+「きゃああっ！　み、見ないでください！」="Oh my gosh! Don't Look!"
+「その……急いでいるので失礼します」="Um... sorry for being in a rush."
+「……なんですか？」="...What is it?"
+「ご、ごめんなさい……今、友達に呼ばれていて……し、失礼します」="Sorry, my friend's calling me. Excuse me."
+「すー……すー……」="Oh... cool."
+「えっ！？　……あ。そ、そっち向いてください！」="Hey! ...What are you looking at!"
+「うぅ、あの、あんまり見ないでください」="Well, don't stare so much."
+「あ、どうも……あなたも、屋上が好きなんですか？」="Oh, thanks... do you also like the rooftop?"
+「あ、あなたも本を借りに来たんですか？」="Oh, are you also here to borrow a book?"
+「あ、あれ？　き、奇遇ですね」="Oh, really? That's crazy."
+「えっと……何をするかも、着くまでナイショなんですか？」="Well... what do you do for the meantime?"
+「あの、さすがにそこまでは……」="Well, that's it..."
+「い、いつも話しかけて来てくれて、嬉しいです」="Thanks for always talking with me."
+「あ、あの、私とで良ければ……な、なんでもありません！」="Oh, well... it's fine with me... Eh! Forget what I said!"
+「あっ、どうも。えっと、見学ですか？」="Oh, is it a tour?"
+「あ、あれ？　今日はコイカツ部もここで活動を……？」="Oh, that? We have koikatsu club here too...?"
```

Comment:
Instead of "We have koikatsu club here too...?", she's more so asking if you have club today